changecalculator/src/main/java/org/msstate/cse/changecalculator/changeproneness/ChangePronenessManager.java

```diff
@@ -0,0 +1,154 @@
+package org.msstate.cse.changecalculator.changeproneness;
+

```

Comment:
There isn't a header file, nor are there adequate comments to explain the file's functionality.